core/src/navigation/TopNav.html

```diff
@@ -1,12 +1,17 @@
 <svelte:window on:click="{closeAllDropdowns}" on:blur="{closeAllDropdowns}"/>
 {#if showTopNav}
-<div class="fd-shellbar {hideNavComponent ? 'hideNavComponent' : ''}">
+<div
+  class="fd-shellbar {hideNavComponent ? 'hideNavComponent' : ''}"
+  tabindex="0"
+>
   <div class="fd-shellbar__group fd-shellbar__group--product">
     {#if responsiveNavSetting==="simple" || responsiveNavSetting==="simpleMobileOnly" || responsiveNavSetting==="Fiori3"}
-    <span
-      class="sap-icon--menu2 lui-burger"
+    <button
+      class="fd-shellbar__button fd-button sap-icon--menu2 lui-burger"
+      aria-controls="0AcWE812"
```

Comment:
Just curious, how does `0AcWE812` come from?

Reply:
@stanleychh have no idea, from the git log I see it was added by @JohannesDoberer  6 months ago, or maybe it was a merge from another branch.

Reply:
@there is no ID bonded to that aria-controls so I removed 2 of them from the TopNav.html 
@stanleychh please check and approve the changes if all is good :)